### PR TITLE
add entities

### LINKED ISSUE #4: Criar a entidade Seller
- [x] Um seller deve ser um usuário
- [x] Um seller deve ter um nome
- [x] Um seller deve ter uma senha
- [x] Um seller deve ter um UserType
- [x] Um seller deve ter uma lista de produtos

### LINKED ISSUE #11: Criar entidade representante
- [x] deve extender User

- [x] deve ter o método de fazer pedido de entrada

### PR DESCRIPTION
**Pull request by DouglasCorreiaMeli**
this PR creates entity seller and agent and closes issues:
#11
 #4

### CODE REVIEW

File: src/main/java/com/mercadolibre/dambetan01/model/User.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        this.password = password;
+    }
+
+    public List <UserType> getUserType() {
+        return userType;
+    }
+
+    public void setUserType(List <UserType> userType) {
+        this.userType = userType;
+    }
```
Comment (warleyvods-meli) [inline src/main/java/com/mercadolibre/dambetan01/model/User.java]: Lombok com @Data

**Review by warleyvods-meli — CHANGES_REQUESTED** [review]
Teste comentario

**Review by rogsilva-meli — APPROVED** [review]
Approved

File: src/main/java/com/mercadolibre/dambetan01/model/User.java (hunk 1 above)
Comment (DouglasCorreiaMeli) [reply src/main/java/com/mercadolibre/dambetan01/model/User.java]: alterado

**Review by warleyvods-meli — APPROVED** [review]
OK!

**Pull request merged**